test_accounts/health_test_accounts.md

```diff
@@ -11,7 +11,9 @@ Use these test accounts to log into the ID.me sandbox environment that is connec
 ---
 
 **NOTE:**
-Details about Conditions associated to a patient can be found in [Test Patient Conditions](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/ramosan-add-patient-conditions/test_accounts/health_test_account_conditions.xlsx)
+
+ * Details about Conditions associated to a patient can be found in [Test Patient Conditions](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/ramosan-add-patient-conditions/test_accounts/health_test_account_conditions.xlsx).
+ * Details about patient with COVID-19 related test data [below](#covid19-test-data).
```

Comment:
I would change this sentence to "Details about a patient with COVID-19 related test data can be found [below]."

Reply:
Since the information is actually across multiple distinct patients, I decided to just pluralize "patient" to "patients instead of "about a patient". Resulting in "Details about patients with COVID-19 related test data can be found [below]."

Reply:
Looks good to me!